recipe/meta.yaml

```diff
@@ -56,6 +57,7 @@ outputs:
         - ipykernel >=6.0.0
         # tqdm extra
         - tqdm >=4.7.4,<=5.0.0dev
+        # skipping bigquery_v2 extra until there is a request to add it
```

Comment:
@conda-forge/google-cloud-bigquery, should we add this extra?
https://github.com/googleapis/python-bigquery/blob/330fa5e3f9a50ccb19441ae40aad191365b0a110/setup.py#L82-L85

Reply:
If not, we need to remove these import tests:
```
        - google.cloud.bigquery_v2
        - google.cloud.bigquery_v2.types
```